src/main/java/com/szymon/domain/Book.java

```diff
@@ -0,0 +1,37 @@
+package com.szymon.domain;
+
+/**
+ * Created by szymon.nowak on 04.04.2016.
+ */
+public class Book {
```

Comment:
Think about book entity:
1) what fields are mandatory in book?
2) can book exist without title and autor?

This book should contain content as simple text.

Check what final in java is.
Check lombok project please.
Check what immutable objects in java are.